src/java.base/share/classes/java/util/zip/ZipOutputStream.java

```diff
@@ -182,6 +182,21 @@ public void setLevel(int level) {
      * The default compression method will be used if no compression method
      * was specified for the entry, and the current time will be used if
      * the entry has no set modification time.
+     *
+     * The zip file format does not record the compression level used for the
```

Comment:
"zip" ->"ZIP" for consistency'

Should this refer to the Zip File Format specification?